devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -394,6 +403,13 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 		}
 		result << createEvent([name: "switch", value: isActive ? "on" : "off", displayed: true])
 		result << createEvent([name: "alarm", value: isActive ? "both" : "off", displayed: true])
+	} else if (cmd.notificationType == 0x07) { //Tamper Alert
+		switch (cmd.event) {
+			case 0x03:
+				result << createEvent([name: "tamper", value: "detected"])
+				result << createEvent([name: "alarm", value: "both"])
```

Comment:
The way I interpreted the table you posted is that the device will send an event parameter when tamper is activated, but will omit it when the tamper switch has been released (cleared).

Is that what the device sends?

Reply:
Yes, it is

Reply:
Then the event with the payload should cause tamper detected and the event with no payload should cause tamper clear.

Reply:
So there is only an option to clear tamper by off event. What should we do in this situation?

Reply:
You can leave the current tamper clear in the "off" event, but I think you should be handling both of the tamper events the device claims to send in the documentation you showed me.